<ui version="4.0" >
 <author>William Pye</author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Settings</class>
 <widget class="QDialog" name="Settings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Vbrfix Settings</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Fix Settings</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>8</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="keepLameInfo" >
          <property name="text" >
           <string>Keep Lame Info</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="alwaysSkip" >
          <property name="text" >
           <string>Always Skip (For testing / Information) - No output</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3_2_2_2_2" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Skip files less than&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="minPercentUnderstood" >
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="value" >
             <number>99</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">% understood&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Output Settings</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>8</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="promptForEach" >
          <property name="text" >
           <string>Prompt For Each</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="replaceOriginals" >
          <property name="text" >
           <string>Replace Originals</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="appendVbrfixed" >
          <property name="text" >
           <string>Append "(vbrfixed)" to output filename</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="outputToDir" >
            <property name="text" >
             <string>Output To Directory :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="outputDir" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browseOutputDir" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="defaultButton" >
       <property name="text" >
        <string>Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>91</width>
         <height>471</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>368</x>
     <y>9</y>
     <width>353</width>
     <height>620</height>
    </rect>
   </property>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>outputToDir</sender>
   <signal>toggled(bool)</signal>
   <receiver>outputDir</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>74</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel" >
     <x>251</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outputToDir</sender>
   <signal>toggled(bool)</signal>
   <receiver>browseOutputDir</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>51</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel" >
     <x>328</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
